apps/server/src/schemas/integration.schema.ts

```diff
@@ -0,0 +1,18 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { HydratedDocument } from 'mongoose';
+
+export type IntegrationDocument = HydratedDocument<Integration>;
+
+@Schema()
+export class Integration {
```

Comment:
I've personal doubts about the purpose of this scheme. I think we will use this integration abstracted by the cloud provider library from azure or aws. What do you think?

Reply:
We decided to use this interface and change in further as necessary.